src/Lib/Validate/Floats.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace tantrum_elastic\Lib\Validate;
+
+use tantrum_elastic\Exception\Validation;
+
+trait Floats
+{
+
+    /**
+     * Validates that the supplied value is a floating point number
+     * @param $value
+     * @param string $message
+     * @param string $exceptionClass
+     * @return bool
+     */
+    public function validateFloat($value, $message = 'Value is not a float', $exceptionClass = 'InvalidFloat')
```

Comment:
What's the advantage of supplying with a specific exception class in here rather than using a specific one?